changelog/releases/mainnet/v1.0.2:1/addresses.json

```diff
@@ -0,0 +1,149 @@
+				"trancheFactory": "0x62F161BF3692E4015BefB05A03a94A40f520d1c0",
+				"ptPool": {
+					"address": "0xce16E7ed7654a3453E8FaF748f2c82E57069278f",
+					"poolId": "0xce16e7ed7654a3453e8faf748f2c82e57069278f00020000000000000000006d",
+					"fee": "0.1",
+					"timeStretch": 26.1
+				},
+				"ytPool": {
+					"address": "0xD5D7bc115B32ad1449C6D0083E43C87be95F2809",
+					"poolId": "0xd5d7bc115b32ad1449c6d0083e43c87be95f280900020000000000000000006c",
+					"fee": "0.003"
+				},
+				"weightedPoolFactory": "0x8E9aa87E45e92bad84D5F8DD1bff34Fb92637dE9",
+				"convergentCurvePoolFactory": "0xb7561f547F3207eDb42A6AfA42170Cd47ADD17BD"
+			}
+		],
+		"lusd3crv-f": [
+			{
+				"address": "0x9b44Ed798a10Df31dee52C5256Dcb4754BCf097E",
+				"expiration": 1632834462,
+				"trancheFactory": "0x62F161BF3692E4015BefB05A03a94A40f520d1c0",
+				"ptPool": {
+					"address": "0xA8D4433BAdAa1A35506804B43657B0694deA928d",
+					"poolId": "0xa8d4433badaa1a35506804b43657b0694dea928d00020000000000000000005e",
+					"fee": "0.1",
+					"timeStretch": 16
+				},
+				"ytPool": {
+					"address": "0xDe620bb8BE43ee54d7aa73f8E99A7409Fe511084",
+					"poolId": "0xde620bb8be43ee54d7aa73f8e99a7409fe51108400020000000000000000005d",
+					"fee": "0.003"
+				},
+				"weightedPoolFactory": "0x8E9aa87E45e92bad84D5F8DD1bff34Fb92637dE9",
+				"convergentCurvePoolFactory": "0xb7561f547F3207eDb42A6AfA42170Cd47ADD17BD"
+			},
+			{
+				"expiration": 1640620258,
+				"address": "0xa2b3d083AA1eaa8453BfB477f062A208Ed85cBBF",
+				"trancheFactory": "0x62F161BF3692E4015BefB05A03a94A40f520d1c0",
+				"ptPool": {
+					"address": "0x893B30574BF183d69413717f30b17062eC9DFD8b",
+					"poolId": "0x893b30574bf183d69413717f30b17062ec9dfd8b000200000000000000000061",
+					"fee": "0.1",
+					"timeStretch": 24
+				},
+				"ytPool": {
+					"address": "0x67F8FCb9D3c463da05DE1392EfDbB2A87F8599Ea",
+					"poolId": "0x67f8fcb9d3c463da05de1392efdbb2a87f8599ea000200000000000000000060",
+					"fee": "0.003"
+				},
+				"weightedPoolFactory": "0x8E9aa87E45e92bad84D5F8DD1bff34Fb92637dE9",
+				"convergentCurvePoolFactory": "0xb7561f547F3207eDb42A6AfA42170Cd47ADD17BD"
+			}
+		],
+		"crvtricrypto": [
+			{
+				"expiration": 1628997564,
+				"address": "0x237535Da7e2f0aBa1b68262ABCf7C4e60B42600C",
+				"trancheFactory": "0x62F161BF3692E4015BefB05A03a94A40f520d1c0",
+				"ptPool": {
+					"address": "0x3A693EB97b500008d4Bb6258906f7Bbca1D09Cc5",
+					"poolId": "0x3a693eb97b500008d4bb6258906f7bbca1d09cc5000200000000000000000065",
+					"fee": "0.1",
+					"timeStretch": 6.16
+				},
+				"ytPool": {
+					"address": "0xF94A7Df264A2ec8bCEef2cFE54d7cA3f6C6DFC7a",
+					"poolId": "0xf94a7df264a2ec8bceef2cfe54d7ca3f6c6dfc7a000200000000000000000064",
+					"fee": "0.003"
+				},
+				"weightedPoolFactory": "0x8E9aa87E45e92bad84D5F8DD1bff34Fb92637dE9",
+				"convergentCurvePoolFactory": "0xb7561f547F3207eDb42A6AfA42170Cd47ADD17BD"
+			}
+		],
+		"dai": [
+			{
+				"expiration": 1702000550,
```

Comment:
same here... delete this tranche since it's a no go

Reply:
Good catch, removed it!